quizzes/quiz_ch3.md

```diff
@@ -0,0 +1,40 @@
+
+{quiz, id: components, attempts: 10}
+
+## Components of reproducibility
+
+? What does "reproducibility on a continuum" mean?
+
+a) Reproducibility that can only be achieved with perfect execution
+B) Reproducibility that needs iterative work to improve
+c) Reproducibility that is dependent on the researcher's expertise
+d) Reproducibility that is achievable only for certain types of analyses
+
+? Which of the following is not a component of reproducibility?
+
+a) Transparency
+b) Accessibility
+c) Consistency
+D) Efficiency
+
+? What does "open source" mean in the context of reproducibility?
+
+A)  Making data and code available for anyone to access and modify
```

Comment:
```suggestion
A) Making data and code available for anyone to access and modify
```

Reply:
"and modify" sounds like random people might modify your code - maybe we mean to access and use? just wondering if we could phrase it a bit differently. I know you mean code review, but just thinking for people new to the subject that this might sound concerning

Reply:
How's this?

Reply:
looks good!